README.md

```diff
@@ -32,40 +38,47 @@ Heroku CNBs are written with the app developer in mind. We want it to be as easy
 
 ## See it in action
 
-In this section we will use Heroku's Buildpacks to transform a Node.js app into a container image. The process for other languages is identical, you are encouraged to try it with your own app!
+This section will use Heroku's Buildpacks to transform a Node.js app into a container image. The process for other languages is identical; you are encouraged to try it with your application!
 
 ### Prerequisites
 
 #### Docker
+
 If you don't have it already, you need to install Docker. Refer to the Docker documentation on how to install it for your operating system: https://docs.docker.com/engine/install/
 
 #### pack
 
-To run our buildpack locally, we will use `pack`, a tool maintained by the Cloud Native Buildpacks project to support the use of buildpacks. Find their documentation about installing it here: https://buildpacks.io/docs/install-pack/
+To run our buildpack locally, we will use `pack`, a tool maintained by the Cloud Native Buildpacks project. Follow their documentation to [install the pack CLI](https://buildpacks.io/docs/install-pack/).
 
 ### Create the app image
 
-First, we need an app to transform into an image. We use [Heroku's "getting started" app for Node.js](https://github.com/heroku/node-js-getting-started), but any other app that uses a supported ecosystem will work. We can clone the repository like so: `git clone https://github.com/heroku/node-js-getting-started.git`. 
+> [!NOTE]
+> For a full hands-on getting started experience [follow our getting started tutorial](TUTORIAL.md).
 
-Then, we can run the following command to build our app image - no additional steps needed! The required buildpacks for your app will be detected automatically.
+First, we need an application to produce an image. We will use [Heroku's getting started app for Node.js](https://github.com/heroku/node-js-getting-started), but any other app that uses a supported ecosystem will work. Clone the application:
 
 ```
-pack build my-app-image --builder heroku/builder:22 --path node-js-getting-started
+$ git clone https://github.com/heroku/node-js-getting-started.git
+$ cd node-js-getting-started
 ```
 
-### Tutorial
+In your application directory, run the `pack build` command. No additional steps are needed! The required buildpacks for your app will be detected automatically:
 
-Check out our [hands-on tutorial](TUTORIAL.md) to learn more about how our buildpacks can help you create production-ready container images.
+```
+$ pack build my-app-image --builder heroku/builder:22 --path .
```

Comment:
Might as well drop `--path .` if we've already changed directory to `node-js-getting-started`

```suggestion
$ pack build my-app-image --builder heroku/builder:22
```

Reply:
From experience, people will run this in the wrong directory. Either they won't CD, or maybe they cloned it into a different directory, or a thousand other problems. The way to correct for this in a tutorial is to give a "check" step (as I did in the tutorial by showing the contents of `ls`) but this is a minimal example. So if this command fails  I want the command to act as a bit of "oh, wait...I'm passing in arguments, is the path correct?" marker.

As a much weaker argument, I've worked with tools that cannot handle a path to a different directory; you can ONLY use them in the current directory. I feel the three arguments here—image name, builder name, and path—are likely the three most common, and I would like to demonstrate them.

As a side note: I would actually like to encourage people to set heroku as the default builder as I do in the tutorial. However, I like that it's explicit here. We would do well to add a "Note" to this section asking people to set their default builder, and also updating the tutorial to show people how they can use `--path`.

Reply:
Fine by me, really - though I'd argue that if you wanted to show people how they can use `--path` in a way that isn't unnecessary, the existing text already did just that (there's no need to `cd` into the directory, as we don't do anything else with the app in this short example):

```
git clone https://github.com/heroku/node-js-getting-started.git

pack build my-app-image --builder heroku/builder:22 --path node-js-getting-started
```

Reply:
I still think the `git clone` command can be presented better though (I moved it to a code block/it's own line here https://github.com/heroku/buildpacks/pull/13/commits/aca76c051f7fb5c1f4775bd1c83fe46771560762, but closed that PR to not conflict with your changes here :))

Reply:
For this case I like the `git clone` without the `cd` and keeping the `--path` (effectively what it is on main but `git clone` is on it's own line.